tower-http/Cargo.toml

```diff
@@ -25,7 +25,8 @@ tower-service = "0.3"
 async-compression = { version = "0.3", optional = true, features = ["tokio"] }
 bytes = { version = "1", optional = true }
 hyper = { version = "0.14", optional = true, default_features = false, features = ["stream"] }
-tokio-util = { version = "0.6", optional = true, default_features = false, features = ["io", "codec"] }
+tokio-util = { version = "0.6", optional = true, default_features = false, features = ["io"] }
+tokio = { version = "1", optional = true, default_features = false }
```

Comment:
I thought tokio didn't bring in any default features? but I guess this doesn't hurt

Reply:
Ah yeah true. Habit but yeah shouldn't hurt.